src/test/scala/io/iohk/ethereum/jsonrpc/EthServiceSpec.scala

```diff
@@ -725,6 +824,40 @@ class EthServiceSpec
     response.transactionResponse shouldBe None
   }
 
+  it should "getRawTransactionByBlockNumberAndIndexRequest return transaction by index" in new TestSetup {
```

Comment:
I think there should be tests for jsonrpc controller added as well (they are for ensuring service is properly wired and json codecs are behaving as expected)

Reply:
Tests for Json RPC controller added:
* eth_getRawTransactionByHash: https://github.com/input-output-hk/mantis/pull/720/commits/f4e5a8922651af2a4189904b7845114d4c3a5358
* eth_getRawTransactionByBlockNumberAndIndex: https://github.com/input-output-hk/mantis/pull/720/commits/fe8fe23f40b223f99975f2f2ea070c31b6316971
* eth_getRawTransactionByBlockHashAndIndex https://github.com/input-output-hk/mantis/pull/720/commits/22b93f2d5c9f22ae1a65a971032416fe2f63eaa8